src/components/cardbuilder/cardBuilder.js

```diff
@@ -1207,7 +1204,28 @@ function buildCard(index, item, apiClient, options) {
         additionalCardContent += getHoverMenuHtml(item, action);
     }
 
-    return '<' + tagName + ' data-index="' + index + '"' + timerAttributes + actionAttribute + ' data-isfolder="' + (item.IsFolder || false) + '" data-serverid="' + (item.ServerId || options.serverId) + '" data-id="' + (item.Id || item.ItemId) + '" data-type="' + item.Type + '"' + mediaTypeData + collectionTypeData + channelIdData + pathData + positionTicksData + collectionIdData + playlistIdData + contextData + parentIdData + startDate + endDate + ' data-prefix="' + escapeHtml(prefix) + '" class="' + className + '"' + ariaLabelAttribute + '>' + cardImageContainerOpen + innerCardFooter + cardImageContainerClose + overlayButtons + additionalCardContent + cardScalableClose + outerCardFooter + cardBoxClose + '</' + tagName + '>';
+    const itemData =
+        ' data-index="' + index + '"' + timerAttributes + actionAttribute
+        + ' data-isfolder="' + (item.IsFolder || false) + '"'
+        + ' data-serverid="' + (item.ServerId || options.serverId) + '"'
+        + ' data-id="' + (item.Id || item.ItemId) + '"'
+        + ' data-type="' + item.Type + '"'
+            + mediaTypeData
+            + collectionTypeData
+            + channelIdData
+            + pathData
+            + positionTicksData
+            + collectionIdData
+            + playlistIdData
+            + contextData
+            + parentIdData
+            + startDate
+            + endDate
+        + ' data-prefix="' + escapeHtml(prefix) + '"';
+
+    cardImageContainerOpen = `<div ${itemData} class="${cardBoxClass}">${cardImageContainerOpen}`;
```

Comment:
Item data has been moved from `card` to `cardBox`. I wonder if it is still working.

Reply:
Indeed the TV layout isn't working as of now. I have marked this PR as a Draft.

I did fix some issues but can't get right clicking to work for now.